README.md

```diff
@@ -13,36 +19,34 @@ Make sure your application has been set up to use go modules (there should be a
 Add the driver with:
 
 ```shell
-go get github.com/neo4j/neo4j-go-driver/v5@<the 5.x tag>
+go get github.com/neo4j/neo4j-go-driver/v5
 ```
 
 #### 4.x
 
 Add the driver with:
 
 ```shell
-go get github.com/neo4j/neo4j-go-driver/v4@<the 4.x tag>
+go get github.com/neo4j/neo4j-go-driver/v4
```

Comment:
Why was the `@<the 4.x tag>` removed? Is this to make people use the lastest version of 4 and 5?

Reply:
It will use the latest tag. Tags have precedence over arbitrary SHAs.